BanchoBot/functions/tournaments/matchup/getStageMods.ts

```diff
@@ -0,0 +1,7 @@
+import { BanchoMod, BanchoMods } from "bancho.js";
+
+export default function getStageMods (modEnum: number | null | undefined): BanchoMod[] {
+    const modParse = BanchoMods.parseBitFlags(modEnum || 0, true).concat(BanchoMods.NoFail);
+    
+    return modParse.concat(BanchoMods.NoFail).filter((v, i, a) => a.findIndex(m => m.enumValue === v.enumValue) === i);
```

Comment:
Uhhh so before applying a suggestion regarding this the intent is basically it's to add nofail as the default mod given in every circumstance and in case the user provided nofail as one of the pool slot's allowed mods then it filters it

Reply:
the point of that suggestion is to not concat nofail twice

Reply:
oh im blind